idd/Energy+.idd.in

```diff
@@ -19458,6 +19458,7 @@ RoomAir:Node:AirflowNetwork:InternalGains,
       \key ElectricLoadCenter:Inverter:LookUpTable
       \key ElectricLoadCenter:Storage:Battery
       \key ElectricLoadCenter:Storage:Simple
+      \key ElectricLoadCenter:Storage:Converter
```

Comment:
Thanks for writing back.  No, we pulled this out and will merge it in right after release.  No worries this time.